### DISCUSSION TITLE
Why LYVE_DUPLEAFLIST and some other vecode have been remove in libyang2? And how should I know the specific error code?

### DISCUSSION THREAD

**Discussion by lyzliyuzhi91** [CESNET/libyang#2021] [Q&A]
I have implemented a NETCONF server, and when it parse XML/JSON data by libyang1, libyang1 would report such vecode code, and I will transform the libyang vecode to error-tag needed by netconf. 
For example, if libyang1 report LYVE_DUPLEAFLIST , and then I will transform it to "data-exists" error-tag, and return such error-tag in error reply.
But now, I cannot get such detailed error code by libyang2, and It seems that I have no other convinient way to get a correct error-tag ?

**Comment by michalvasko** [comment]
We have faced the same problem in *netopeer2* but it is simply not worth it to have all these specific validation codes. The way we implemented it is based on the specific (part of) error message.
